src/plugins/uploader.js

```diff
@@ -115,8 +115,6 @@ class Uploader {
 		// so that browsers can play them correctly
 		if (detectedMimeType === "audio/vnd.wave") {
 			detectedMimeType = "audio/wav";
-		} else if (detectedMimeType === "audio/x-flac") {
```

Comment:
This is done because the `file-type` module returns `audio/x-flac` rather than `audio/flac`. This sets it to `audio/flac` instead, which I think is generally what you're trying to do.

Removing this seems like it's counterproductive. Unless you noticed that the newer versions of `file-type` return `audio/flac` now?

Reply:
I wonder if this `file-type` plugin is the one responsible for #4429? 🤔